bookshelter-app/src/components/CategoriesSection/index.tsx

```diff
@@ -5,31 +5,41 @@ import { Category } from '../../types';
 import { useEffect, useState } from 'react';
 import { get } from '../../services/api';
 import { API } from '../../constants/api';
+import { useToast } from '../../hooks/useToast';
 
 const CategoriesSection = () => {
   const [categories, setCategories] = useState<Category[]>([]);
   const { slug, id } = useParams<{ slug: string; id: string }>();
   const [searchParams] = useSearchParams();
+  const { showToast } = useToast();
 
   const name = searchParams.get('query');
   const location = useLocation();
   const isHomePage = location.pathname === '/';
 
+  console.log('bbb');
```

Comment:
remove log

Reply:
For the next PR, pls review code changed before submitting

Reply:
Fixed!